terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1669,11 +1674,26 @@ def qq(*words):
                                         }
                                     }
                                 }
-                            }, indent=4)
+                                # FIXME: Hitting user_data size limits
```

Comment:
```suggestion
                                # FIXME: Re-enable formatting of the JSON above
```

because it is more specific about what needs to be done *here*. The reason for all of this is evident from the issue.